package.json

```diff
@@ -23,7 +23,7 @@
     "@types/mocha": ">=9.1.0",
     "@types/node": ">=12.0.0",
     "@zetachain/addresses-tools": "^0.0.7",
-    "axios": "^1.3.6",
+    "@zetachain/faucet-cli": "^1.3.0",
```

Comment:
Please wait until we have it reviewed, stable and pointing to production faucet.

Reply:
Yeah, of course 👍 just wanted to test it out asap. I'll convert this PR into a draft.